aten/src/ATen/Context.cpp

```diff
@@ -352,6 +354,10 @@ bool Context::hasMKLDNN() {
 #endif
 }
 
+bool Context::hasKleidiAI() {
+  return AT_KLEIDIAI_ENABLED();
```

Comment:
given the kernels are pretty much I8MM based, it would be better to check for aarch64 cpus runtime cpuinfo checks or compiler FEATURE flags for I8MM.

Reply:
Not Required